src/nodes/dir.rs

```diff
@@ -16,7 +16,10 @@ extern crate fuse;
 extern crate libc;
 extern crate time;
 
+use super::conv;
```

Comment:
Is there a way to make this crate-relative instead of module-relative?

Reply:
Probably. For now I've reordered this chunk alphabetically, which collates this new `use` with other crate-relative `use`s. I notice there are several more instances of this already in other files, so I'd rather fix it in one go in a separate change.